cli/lib/kontena/cli/cloud/login_command.rb

```diff
@@ -51,6 +55,37 @@ def finish
       true
     end
 
+    def remote_login
+      client_id = kontena_account.client_id || Kontena::Client::CLIENT_ID
+      params = {
+        client_id: client_id
+      }
+      cloud_url = kontena_account.url
+      client = Kontena::Client.new(cloud_url, nil)
+      auth_request_response = client.post('/auth_requests', {}, params)
+      verification_uri = URI.parse(auth_request_response['verification_uri'])
+      puts "Please visit #{verification_uri.to_s.colorize(:cyan)} and enter the code"
+      puts
+      puts "#{auth_request_response['user_code']}"
+      puts
+      puts "Once the authentication is complete you can close the browser"
+      puts "window or tab and return to this window to continue."
+      puts
+
+      code_request_params = {
+        client_id: client_id,
+        device_code: auth_request_response['device_code']
+      }
+      code_response = nil
+      spinner "Waiting for authentication" do
+        until code_response do
+          code_response = client.post("/auth_requests/code", {}, code_request_params) rescue nil
```

Comment:
instead of rescuing with nil, you could add expectations like in master login:

```ruby
        response = client.request(
          http_method: :get,
          path: authentication_path(auth_params),
          expects: [501, 400, 302, 403],
          auth: false
        )

        if client.last_response.status == 302
          ...
        end
```